README.md

```diff
@@ -4,12 +4,12 @@ This package provides AngelList's `.eslintrc.js` as an extensible shared config.
 
 ## Usage
 
-Our default export contains all of our ESLint rules, including ECMAScript 6+, React, TSLint compat bindings, presets inherited, etc. Thus, there are a ton of peer dependencies to satisfy.
+Our default export contains all of our ESLint rules, including ECMAScript 6+, React, presets inherited, etc. Thus, there are a ton of peer dependencies to satisfy.
 
 ### 1. Install config as dependency
 
 ```sh
-yarn add --dev 'git+https://github.com/venturehacks/eslint-config-angellist#0.3.1'
+yarn add --dev 'git+https://github.com/venturehacks/eslint-config-angellist#1.0.0'
```

Comment:
Is there a 1.0.0? Checking [its tags](https://github.com/venturehacks/eslint-config-angellist/tags) I can only see up to 0.5.1 🤔

Reply:
There is no 1.0, but there will be ;)